### ISSUE TITLE
ValueError: Cannot find twitter account with name = realDonaldTrump

### ISSUE THREAD

**Issue by crazyZSShuo** [twintproject/twint#763]
File "E:/TwitterScrapy/twint/twitter_spider.py", line 9, in <module>
    twint.run.Search(c)
  File "E:\TwitterScrapy\twint\twint\run.py", line 329, in Search
    run(config, callback)
  File "E:\TwitterScrapy\twint\twint\run.py", line 228, in run
    get_event_loop().run_until_complete(Twint(config).main(callback))
  File "c:\users\10848\appdata\local\programs\python\python37\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "E:\TwitterScrapy\twint\twint\run.py", line 154, in main
    await task
  File "E:\TwitterScrapy\twint\twint\run.py", line 171, in run
    raise ValueError("Cannot find twitter account with name = " + self.config.Username)

**Comment by elamje** [comment]
I'm getting the same issue with various accounts as well!

**Comment by evawxiao** [comment]
Me too

**Comment by caspx** [comment]
Here as well. 
But it only occurs when installing twint from the git repo 
(i.e. pip3 install . -r requirements.txt from the repo directory) 
The issue does not occur when installing from pip (i.e. pip3 install twint)

**Comment by evawxiao** [comment]
> Here as well.
> But it only occurs when installing twint from the git repo
> (i.e. pip3 install . -r requirements.txt from the repo directory)
> The issue does not occur when installing from pip (i.e. pip3 install twint)

Thanks for this! I was able to solve the issue by removing twint and reinstalling it with pip.

**Comment by elamje** [comment]
Yeah, I figured out the issue. When you install from pip, it copies the repo to your /bin/ folder to allow you to run the "twint" command line function. I was trying to change the code inside of my personal dev/twint directory rather than the bin/twint directory that the "twint" command actually uses.
